packages/components/psammead-story-promo/src/TextStyles.jsx

```diff
@@ -104,6 +129,9 @@ const TextGridItem = styled.div`
       padding: initial;
       ${({ promoType }) => textGridStyles[promoType]}
     }
+
+    ${({ promoType }) =>
```

Comment:
The headline already has padding-bottom applied to it, so I targeted the timestamp specifically to add padding to it to prevent double padding being applied when there is no timestamp. Initially I wanted to use the `:last-child` selector to apply it ensuring there's always 8px of padding between the text and image, but I couldn't get that to work.

In terms of the story, good point. I'd forgotten that conversation had happened. I can absolutely see why building complex stories is not wanted, in this case I thought it would be the easiest way to check I was adding it properly. I'll remove it now.

Reply:
Since we won't have a story showing how the padding-bottom applied to the `time` looks like, do you think it would be useful to add a short comment above the `leadingPromoTimestampPadding` or is it auto explanatory?

Reply:
A comment is a good idea - it's not clear even with the story why it's applied only to the `timestamp`.